src/main/java/de/wi2020sebgroup1/cinema/controller/ShowController.java

```diff
@@ -102,8 +102,9 @@
 			try {
 				CinemaRoom room = roomSearch.get();
 				toAdd.setCinemaRoom(room);
+				Optional<CinemaRoomSeatingPlan> seatingPlanSearch = seatingPlanRepository.findById(room.getCinemaRoomSeatingPlan().getId());
```

Comment:
Unnötiger Aufwand => Doppelte Anfrage, welche in einer Funktioniert. 
Hier bitte Rückgängig

Reply:
Dann kann das try/catch auch weg